core/storage/user/gae_models.py

```diff
@@ -23,7 +23,8 @@
 from google.appengine.datastore import datastore_query
 from google.appengine.ext import ndb
 
-(base_models,) = models.Registry.import_models([models.NAMES.base_model])
+(base_models, suggestion_models) = models.Registry.import_models(
```

Comment:
I'm not sure that we want to encourage model modules to be tightly coupled; could we maybe instead find a common place to store variables like ``MINIMUM_SCORE_REQUIRED_TO_REVIEW``? Maybe feconf or CONSTANTS?

Reply:
Moved to feconf